Helper/Order.php

```diff
@@ -222,6 +222,21 @@ function (?int $foundScore, \stdClass $fraudAssessment): ?int {
         return $eq8Score;
     }
 
+    /**
+     * Gets local EQ8 scores for a set of order IDs
+     * Does not verify that all order IDs provided were found
+     * @param array $orderIds
+     * @return Collection the found orders
+     */
+    public function getOrderEQ8Scores(array $orderIds): Collection
+    {
+        return $this->orderCollectionFactory->create()
+             ->addFieldToSelect(OrderInterface::ENTITY_ID)
+             ->addFieldToSelect(self::EQ8_SCORE_COL)
+             ->addFieldToFilter(OrderInterface::ENTITY_ID, ['in' => $orderIds])
+             ;
```

Comment:
I legit cannot believe the linter let you put the semicolon here. 😄

Reply:
I stole it from another usage of this factory further down the file 😉